ckanext/s3filestore/plugin.py

```diff
@@ -110,10 +109,9 @@ def after_update_resource_list_update(self, visibility_level, pkg_id, pkg_dict):
                     target_acl=visibility_level)
         LOG.debug("after_update_resource_list_update: Package %s has been updated, notifying resources finished", pkg_id)
 
-    def enqueue_resource_visibility_update_job(self, visibility_level, pkg_id, pkg_dict):
+    def enqueue_resource_visibility_update_job(self, visibility_level, pkg_id):
         ckan_ini_filepath = os.path.abspath(toolkit.config['__file__'])
-        resources = pkg_dict
-        args = [ckan_ini_filepath, visibility_level, pkg_id, resources]
+        args = [ckan_ini_filepath, visibility_level, pkg_id]
         kwargs = {
```

Comment:
Are we going to add any other arguments, eg failure-handling?

Reply:
let me try, it seems there is more args you can call.
#toolkit.enqueue_job(fn, args=None, kwargs=None, title=None, queue=DEFAULT_QUEUE_NAME, rq_kwargs=None)

Reply:
have included, unsure how to test as our other plugins don't do this just yet.